src/common-util/src/main/java/ee/ria/xroad/common/SystemProperties.java

```diff
@@ -174,6 +174,9 @@ private SystemProperties() {
     public static final String SERVER_CONF_CACHE_PERIOD =
             PREFIX + "proxy.server-conf-cache-period";
 
+    public static final String INTERNAL_KEY_CACHE_PERIOD =
+            PREFIX + "proxy.internal-key-cache-period";
```

Comment:
Setting server-conf-cache-period to zero disables all serverconf caching, in which case this setting has no effect. Is it really necessary to have a separate cache period for the internal key?

Reply:
These now use the same cache period: `proxy.server-conf-cache-period` which is 60 seconds by default. Also updated documentation accordingly